src/hangar/checkout.py

```diff
@@ -64,15 +60,15 @@ class ReaderCheckout(object):
     """
 
     def __init__(self,
-                 base_path: os.PathLike, labelenv: lmdb.Environment,
+                 base_path: Path, labelenv: lmdb.Environment,
```

Comment:
it could be both `Path` and `str`?

Reply:
no, this is internal to hangar. We expect a `Path` object.